simulation/src/test/resources/report.md

```diff
@@ -13,14 +13,14 @@
         fast_500s_then_revert[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=100.0%	client_mean=PT0.080925333S 	server_cpu=PT5M3.47S      	client_received=3750/3750	server_resps=3750	codes={200=3750}
             fast_500s_then_revert[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=76.7%	client_mean=PT0.055463644S 	server_cpu=PT3M27.988666346S	client_received=3750/3750	server_resps=3750	codes={200=2876, 500=874}
                       fast_500s_then_revert[UNLIMITED_ROUND_ROBIN].txt:	success=76.7%	client_mean=PT0.055463644S 	server_cpu=PT3M27.988666346S	client_received=3750/3750	server_resps=3750	codes={200=2876, 500=874}
-               live_reloading[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=57.0%	client_mean=PT3.006624019S 	server_cpu=PT1H41M25.16S  	client_received=2500/2500	server_resps=2156	codes={200=1424, 500=732, Failed to make a request=344}
+               live_reloading[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=38.6%	client_mean=PT2.463828397S 	server_cpu=PT46M4.48S     	client_received=2500/2500	server_resps=964	codes={200=964, Failed to make a request=1536}
                    live_reloading[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=53.4%	client_mean=PT3.028114756S 	server_cpu=PT1H40M36.01S  	client_received=2500/2500	server_resps=2139	codes={200=1336, 500=803, Failed to make a request=361}
                              live_reloading[UNLIMITED_ROUND_ROBIN].txt:	success=58.6%	client_mean=PT2.841084S    	server_cpu=PT1H58M22.71S  	client_received=2500/2500	server_resps=2500	codes={200=1465, 500=1035}
           one_big_spike[CONCURRENCY_LIMITER_BLACKLIST_ROUND_ROBIN].txt:	success=79.0%	client_mean=PT1.478050977S 	server_cpu=PT1M59.71393673S	client_received=1000/1000	server_resps=790	codes={200=790, Failed to make a request=210}
-                one_big_spike[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=81.6%	client_mean=PT1.512223628S 	server_cpu=PT2M2.4S       	client_received=1000/1000	server_resps=816	codes={200=816, Failed to make a request=184}
```

Comment:
I'd feel a bit better holding off merging this until the queue is in place given that we have users.